spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/BlasWrapper.scala

```diff
@@ -69,10 +84,82 @@ private[bigdl] class BlasWrapper(val module: AbstractModule[Activity, Activity,
     }).toArray
   }
 
+  /**
+   * Blas layers may not have good performance with mkldnn backend,
```

Comment:
=> Blas layers normally does not have competitive performance when running under mkldnn ?

Reply:
ok, good comments